src/App.tsx

```diff
@@ -1,204 +1,110 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { ErrorMessages, FilterStatus, Todo } from './types/Todo';
+import { Header } from './component/Header';
+import { Footer } from './component/Footer';
+import { ErrorNotification } from './component/ErrorNotification';
+import { TodoList } from './component/TodoList';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorMessages>(
+    ErrorMessages.DEFAULT,
+  );
+  const [loading, setLoading] = useState(true);
```

Comment:
```suggestion
  const [isloading, setIsLoading] = useState(true);
```
why default is true?
better use false as deafault and enable where it needed